### PR TITLE
Search: changelog and README for v1.1.0-beta plugin release

### PR DESCRIPTION
**Pull request by bluefuton**
#### Changes proposed in this Pull Request:
Prepare a changelog and README update for Jetpack Search plugin v1.1.0.

#### Other information:

- [x] Have you written new tests for your changes, if applicable?
- [x] Have you checked the E2E test CI results, and verified that your changes do not break them?

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:
Check that the changelog and README look correct.

### CODE REVIEW

File: projects/plugins/search/readme.txt [hunk 1]
```diff
@@ -111,9 +111,17 @@ You can purchase a Search subscription directly through this plugin or via the [
 5. Manage all of your Jetpack products, including Search, in a single place.
 
 == Changelog ==
-### 1.0.0 - 2022-05-30
-#### Added
-- Initial release.
+### 1.0.1-beta - 2022-07-29
```
Comment (bluefuton) [inline projects/plugins/search/readme.txt]:
```suggestion
### 1.1.0-beta - 2022-07-29
```

File: projects/plugins/search/CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,19 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.0.1-beta] - 2022-07-29
```
Comment (bluefuton) [inline projects/plugins/search/CHANGELOG.md]:
```suggestion
## [1.1.0-beta] - 2022-07-29
```

File: projects/plugins/search/CHANGELOG.md [hunk 3]
```diff
@@ -5,6 +5,19 @@ All notable changes to this project will be documented in this file.
...
+
+### Fixed
+- E2E tests: fixed pretest cleanup script not running
+- Search: update plugin README
+
 ## 1.0.0 - 2022-05-30
 ### Added
 - Initial release.
+
+[1.0.1-beta]: https://github.com/Automattic/jetpack-search-plugin/compare/v1.0.0...v1.0.1-beta
```
Comment (sdixon194) [inline projects/plugins/search/CHANGELOG.md]:
```suggestion
[1.1.0-beta]: https://github.com/Automattic/jetpack-search-plugin/compare/v1.0.0...v1.0.1-beta
```

File: projects/packages/connection/CHANGELOG.md [hunk 4]
```diff
@@ -5,6 +5,8 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.41.8] - 2022-07-29
+
```
Comment (sdixon194) [inline projects/packages/connection/CHANGELOG.md:10]:
```suggestion
- Updated package dependencies

```

**Review by sdixon194 — DISMISSED** [review]
Just two notes!

File: projects/packages/connection/CHANGELOG.md (hunk 4 above)
Comment (bluefuton) [reply projects/packages/connection/CHANGELOG.md:10]: Ah yes - the release process did warn me there was one with no content! Good catch.

File: projects/plugins/search/CHANGELOG.md (hunk 3 above)
Comment (bluefuton) [reply projects/plugins/search/CHANGELOG.md]: Looks like I need to update the version number in the compare URL too.

Comment (bluefuton) [inline projects/plugins/search/CHANGELOG.md]:
```suggestion
[1.1.0-beta]: https://github.com/Automattic/jetpack-search-plugin/compare/v1.0.0...v1.1.0-beta
```

**Review by sdixon194 — APPROVED** [review]

**Comment by bluefuton** [comment]
🚢 r249790-wpcom

**Pull request merged**